client/hotrod-client/src/test/java/org/infinispan/client/hotrod/configuration/ConfigurationTest.java

```diff
@@ -550,6 +561,18 @@ private void validateConfiguration(Configuration configuration) {
       assertEqualsConfig(NearCacheMode.INVALIDATED, NEAR_CACHE_MODE, configuration);
       assertEqualsConfig(10_000, NEAR_CACHE_MAX_ENTRIES, configuration);
       assertEqualsConfig("near.*", NEAR_CACHE_NAME_PATTERN, configuration);
+      assertEquals(2, configuration.clusters().size());
+      for (int i = 0; i < configuration.clusters().size(); i++) {
```

Comment:
I think this code is a bit too complicated for what it tests.
IMHO is better to avoid for loops if we have just 2 properties as it seems. 
```java
 p.setProperty(CLUSTER_PROPERTIES_PREFIX + ".siteA", "hostA1:11222; hostA2:11223");
 p.setProperty(CLUSTER_PROPERTIES_PREFIX + ".siteB", "hostB1:11222; hostB2:11223");
```